### PR TITLE
Add webhook endpoint

### PR DESCRIPTION
**Pull request by plaindocs**
Hey @blaknite could you help me out with missing info here please.

CC @JuanitoFatas 

Fix for #931

### CODE REVIEW

File: pages/apis/rest_api/pipelines.md.erb [hunk 1]
```diff
@@ -563,6 +563,37 @@ Required scope: `write_pipelines`
...
+
+Create webhook for an existing pipeline.
+
+```bash
+curl -X POST "https://api.buildkite.com/v2/organizations/{org.slug}/pipelines/{slug}/webhook"
+```
+
+Required [request body properties](/docs/api#request-body-properties):
+
+<table>
```
Comment (blaknite) [inline pages/apis/rest_api/pipelines.md.erb:576]: We don't need to send any params along with this request. The slugs in the path are all that's required.

**Review by blaknite — COMMENTED** [review]
Thanks for sorting out some docs :)

**Comment by plaindocs** [comment]
@JuanitoFatas @blaknite 

Folks, I'm confused, do we need to send a body with URL and events? Or am I misunderstanding what this does?

File: pages/apis/rest_api/pipelines.md.erb [hunk 2]
```diff
@@ -563,6 +563,26 @@ Required scope: `write_pipelines`
 
 Success response: `204 No Content`
 
+## Add a webhook
+
+Create webhook for an existing pipeline.
```
Comment (yob) [inline pages/apis/rest_api/pipelines.md.erb:568]: Maybe it's worth clarifying what kind of webhook? Is it configuring our system to send a webhook somewhere? Or creating a webhook on GitHub that will ping us on code push? (I think it's this one)

**Pull request merged**